### ISSUE TITLE
Toggle: Fabric Example has bad examples for aria metadata; in fact onAriaLabel and offAriaLabel should not even exist

### ISSUE THREAD

**Issue by philipzloh** [OfficeDev/office-ui-fabric-react#5021]
### Bug Report

- __Package version(s)__: (fill this out)
- __Browser and OS versions__: (fill this out if relevant)

#### Priorities and help requested (not applicable if asking question):

Are you willing to submit a PR to fix? Yes
Requested priority: Normal

Products/sites affected:

https://github.com/OfficeDev/office-ui-fabric-react/pull/3421 introduced a bad example of aria-labels in Toggle. Aria-labels should not carry instructional information -- in fact, there should be no reason why the on label and the off label should be different at all. The on/off-ness should come from the aria-pressed metadata. 

#### Actual behavior:

- onAriaLabel exists
- offAriaLabel exists
- example site has both labels with instructional strings in the labels

#### Expected behavior:

- only one ariaLabel exists
- does not have instructional strings

**Comment by dzearing** [comment]
@cschlechty Do you have thoughts on this?

**Comment by cschlechty** [comment]
I'd agree. For basic controls with well defined behavior we shouldn't be embedding usage strings. The AT can add that, and it is usually a user configurable setting.

**Comment by philipzloh** [comment]
Not just usage strings, we should not have two different aria labels at all!

It should be a role=switch with aria-checked=true|false (ARIA 1.1) or role=checkbox with aria-checked=true|false (ARIA 1.0).

**Comment by dzearing** [comment]
@philipzloh Thank you for taking this on. I think with some minor tweaks your PR looks fine. Thanks for making things better for everyone!
